src/py42/services/cases.py

```diff
@@ -0,0 +1,194 @@
+from collections import OrderedDict
+
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+from py42.settings import items_per_page
+
+
+class CasesService(BaseService):
+    """`Rest documentation https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases`__ ."""
```

Comment:
Should prob not reference Core int

Reply:
As discussed offline, changed link to https://developer.code42.com/api/#tag/Cases